test/unit/sessions/softphone-session-handler.test.ts

```diff
@@ -2035,7 +2046,7 @@ describe('isEndedState()', () => {
   });
 
   it('should return "false" for non-ended states', () => {
-    expect(isEndedState(undefined)).toBe(false);
+    expect(isEndedState(undefined as any)).toBe(false);
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/MyPureCloud/genesys-cloud-webrtc-sdk/pullRequest?prid=9637746)
```suggestion
    expect(isEndedState(undefined as unknown)).toBe(false);
```